src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  color: $main-text-color;
+  background-color: #fff;
+  border: 1px solid $light-grey-color;
+  padding: 32px 16px 16px;
+  transition: $hover-duration;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+    transition: $hover-duration;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey-text-color;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    font-size: 10px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    line-height: 18px;
+
+    &-label {
+      color: $grey-text-color;
+      font-size: 12px;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    box-sizing: border-box;
+    width: 100%;
+    padding: 11px;
+    color: #fff;
+    border-radius: 5px;
+    background-color: $accent-color;
+    border: 1px solid $accent-color;
+    font-size: 14px;
+    font-weight: 700;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    line-height: 16px;
+    transition: all $hover-duration;
+
+    &:hover {
+      background-color: #fff;
+      color: $accent-color;
+    }
+  }
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.2);
```

Comment:
The hover effect scales the card to 1.2 times its size. This might cause layout shifts or overflow issues. Consider using a smaller scale value or adjusting the layout to accommodate this transformation.